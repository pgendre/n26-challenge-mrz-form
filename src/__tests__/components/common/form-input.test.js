import React from 'react'

import Adapter from 'enzyme-adapter-react-16'
import { shallow, configure } from 'enzyme'

import FormInput from '../../../components/common/form-input'

configure({ adapter: new Adapter() })

test('FORM INPUT 01 : Should render component', () => {
  let props = {
    type: 'select',
    options: [
      { label: 'label A', value: 'value A' },
      { label: 'label B', value: 'value B' }
    ],
    onChange: () => null
  }
  const wrapper = shallow(<FormInput {...props} />)
  expect(wrapper).toMatchSnapshot()
})
