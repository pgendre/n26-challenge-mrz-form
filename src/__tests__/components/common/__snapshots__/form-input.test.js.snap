// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FORM INPUT 01 : Should render component 1`] = `ShallowWrapper {}`;
